### ISSUE TITLE
Console errors when entering a search string

### ISSUE THREAD

**Issue by awlunsfo** [mcrowe/soulmate.js#4]
I apologize in advance for my newbie-ness to all of this:

Setup:
Rails 3.1.3
MongoDB through MongoID (2.3.4)

I've mounted Soulmate into my existing rails app and I'm trying to use your plugin to render an auto-completer. So far I can send and retrieve data through Redis with Soulmate, and after following your guide I have functioning auto-complete input box...well, almost functioning.

Whenever I enter a search string into the input box, I get console errors (using Chrome 16.0.912.75 m) to the tune of: 

GET http://127.0.0.1:3000/sm/search?callback=jQuery16209289398335386068_1326164417210&term=doc&types%5B%5D=profession&limit=5 
_f.ajaxTransport.send_           **jquery-1.6.2.min.js:19**
_f.extend.ajax_                         **jquery-1.6.2.min.js:19**
_Soulmate.fetchResults_       **jquery.soulmate.js:266**
_Soulmate.handleKeyup_    **jquery.soulmate.js:243**
___bind_                                  **jquery.soulmate.js:3**
_f.event.handle_                      **jquery-1.6.2.min.js:18**
_f.event.add.i.handle.k_          **jquery-1.6.2.min.js:17**

...and nothing renders where the Suggestion area should.

What's strange (to me) is that I can copy+paste that url from the GET request into my browser and it returns a valid string from  the Soulmate server:

jQuery162031847008573822677_1326164120568({"term":"doc","results":{"profession":[{"term":"Doctor","id":"4f0ba88969dabc2cde00001c"}]}})

I can't tell if the error is coming from jquery.soulmate.js or if something bad is being passed to jquery-1.6.2.min.js...or something totally different.

If you are able to shed any light on this, I would appreciate it.  Thanks :)

**Comment by mcrowe** [comment]
Hey awlunsfo,

No worries. Thanks for trying it out! 

It would be really helpful to know if the soulmate.js specs run for you. Could you clone the repository and open spec/spec_runner.html in your browser? Let me know if there are any failures.

Also, is there any more to the error print out that you get? I see a stack trace there, but can't see the actual error.

Finally, you could try getting the demo to run by opening demo/index.html in your browser and see if that works for you. It may be something in the styles on your page that are keeping the suggestions container from displaying.

Hope I can help get this figured out!

Cheers,
Mitch

**Comment by awlunsfo** [comment]
Hey Mitch,

Thanks for the quick response. I ran the demo and ran spec_runner.html and everything came out great, and after I thought about it a little more, the problem became obvious: I was using an incorrect URL for the soulmate server. Once I corrected that, everything worked like a charm.

That's what I get for staying up and trying to fix it instead of just walking away and coming back to it, haha.

Anyways, thanks for all your help. Your plugin is fantastic :)

-Aaron
